src/LocalistProcessor.php

```diff
@@ -166,6 +174,30 @@ private function create_file_and_array($url) {
   }
 
 
+  private function create_media_from_file($image_array,$media_name) {
+    $media_id = $this->find_media_for_file($image_array);
+    if($media_id) {
+      return $media_id;
+    }
+
+    $media = Media::create([
+      'bundle'           => 'image',
+      'uid'              => 1,
+      'field_media_image' => $image_array,
+    ]);
```

Comment:
Note to add to media field help/description text (and module README I suppose) that the importer expects there to be an image media bundle called "image" with an image file field called "field_media_image".

(Could the be another form field...?  Suuuure...?  But I think it's okay like this, these are pretty standard bundle/field names.  We can always add another form field to control this setting later, if it becomes a problem.)